rtl/eclrtl/rtlformat.cpp

```diff
@@ -1054,7 +1054,7 @@ void CommonCSVWriter::outputBool(bool field, const char* fieldName)
 {
     if (!checkHeaderName(fieldName))
         return;
-    addContentField((field) ? "true" : "false", fieldName);
```

Comment:
I would retain the old function as well as the new function - passing in the string explicitly is a bit error prone.